src/util/newconfig/ConfigFileJson.cpp

```diff
@@ -69,14 +74,17 @@ extractJsonValue(boost::json::value const& jsonValue)
     if (jsonValue.is_double()) {
         return jsonValue.as_double();
     }
+    if (jsonValue.is_null()) {
+        return NullType{};
+    }
     ASSERT(false, "Json is not of type int, uint, string, bool or double");
```

Comment:
maybe add `null` here. "Json is not of type int, uint, string, bool, double or empty")